samples/webhooks.cs

```diff
@@ -13,14 +13,12 @@ class Webhooks
 
         public void Initialise()
         {
-            // Instantiate SDK manager as below.   
-            // You can also optionally pass configurations, logger, etc. 
-            SDKManager sdkManager = SdkManagerBuilder
-                  .Create() // Creates SDK Manager Builder itself.
-                  .Build();
+            // Optionally initialise SDKManager to pass custom configurations, logger, etc. 
+            // SDKManager sdkManager = SdkManagerBuilder.Create().Build();
 
-            // Instantiate WebhooksClient using the created SDK manager
-            webhooksClient = new WebhooksClient(sdkManager);
+            StaticAuthenticationProvider staticAuthenticationProvider = new StaticAuthenticationProvider(token);
+            // Instantiate ModelDerivativeClient using the auth provider
```

Comment:
It should be Instantiate AdminClient pls update.

Reply:
Fixed (It's webhookClient not admin)